apax/layers/readout.py

```diff
@@ -15,13 +15,16 @@ class AtomisticReadout(nn.Module):
     b_init: str = "zeros"
     use_ntk: bool = True
     n_shallow_ensemble: int = 0
+    is_feature_fn: bool = False
     dtype: Any = jnp.float32
 
     def setup(self):
-        readout_unit = [1]
-        if self.n_shallow_ensemble > 0:
-            readout_unit = [self.n_shallow_ensemble]
-        units = [u for u in self.units] + readout_unit
+        units = [u for u in self.units]
```

Comment:
Why not `list(self.units)` and what type is `self.units` that it has to be iterated trough like this?

Reply:
Ahh, I see you make a copy